docs/_articles/en/release-information.md

```diff
@@ -6,6 +6,18 @@ lang: en
 ### To update the plugin, please follow the linked [instructions](./en/getting-started/install/#upgrade-plug-in).
```

Comment:
@rmohan20 You might recall the release notes edits I provided months ago? Similar to that, I'd like us to start following the Salesforce CLI pattern for release notes (example: https://github.com/forcedotcom/cli/blob/main/releasenotes/sfdx/README.md). So we should start prefacing each release note item with NEW/CHANGE/FIX.

Here are my edits for this release - thanks for sharing:

( a couple questions - based on a Google search, I believe JavaScript is the correct way to capitalize that, and for the babel/parser I believe that "@babel/eslint-parser" is also the correct way to display that info. For the second eslint line item, I'm not sure HOW a developer is supposed to identify the files? We should include the command. If this is extra detail and related to the eslint-parser, then it should be one line item, not two)

* NEW:
- Execute the `eslint` engine with `@babel/eslint-parser` to accommodate modern JavaScript syntax
- Identify JavaScript target files that were analyzed by both `eslint` and `eslint-lwc`, and throw warning about the possibility of duplicate violations
* FIX: The survey request banner works correctly.
* FIX: We made some minor code improvements to the Salesforce Graph engine.
* FIX: We updated the RetireJS Vulnerability Repository.
* NEW: The PMD engine is upgraded to 6.48.0.